File: js/programmers/level.1/prgm_133499_1.js
function solution(babbling) {
  let count = 0;

  for (let word of babbling) {
    if (/(aya|ye|woo|ma)\1+/g.test(word)) continue;
    if (/^(aya|ye|woo|ma)+$/g.test(word)) count++;
  }

  return count;
}

File: python/programmers/level.1/prgm_133499_1.py
def solution(babbling):
    count = 0
    arr = ["aya", "ye", "woo", "ma"]

    for i in babbling:
        for j in arr:
            if j * 2 not in i:
                i = i.replace(j, " ")
        
        if i.isspace():
            count += 1

    return count